### PR TITLE
refactor!: refactoring log settings

### PR DESCRIPTION
**Pull request by xxxxxliil**
BREAKING CHANGE: delete environment variables LOG_TO_STDOUT in Dockerfile

### CODE REVIEW

**Comment by xxxxxliil** [comment]
@Magicalex Do we need anything else?

File: Dockerfile [hunk 1]
```diff
@@ -56,13 +56,15 @@ RUN apk add --no-progress --no-cache \
...
-  && rm -rf /flarum/.composer /tmp/* \
+  && rm -rf /flarum/.composer /tmp/* /flarum/app/storage/logs \
+  && cd /flarum/app/storage/ \
+  && ln -s /var/log/flarum logs \
   && setcap CAP_NET_BIND_SERVICE=+eip /usr/sbin/nginx
 
 COPY rootfs /
 RUN chmod +x /usr/local/bin/* /etc/s6.d/*/run /etc/s6.d/.s6-svscan/*
-VOLUME /etc/nginx/flarum /flarum/app/extensions /flarum/app/public/assets /flarum/app/storage/logs
+VOLUME /etc/nginx/flarum /flarum/app/extensions /flarum/app/public/assets /var/log
```
Comment (Hydrog3n) [inline Dockerfile:69]: I would preferre to open only the flarum logs so /var/log/flarum is better no?

Comment (xxxxxliil) [reply Dockerfile:69]: > I would preferre to open only the flarum logs so /var/log/flarum is better no?

But the logs for nginx and php-fpm are also collected here...

Comment (xxxxxliil) [reply Dockerfile:69]: > I would preferre to open only the flarum logs so /var/log/flarum is better no?

I've decided to do it.

**Comment by Magicalex** [comment]
I'm not agree with this PR.
You can add volumes as you want to get the log files.

**Comment by xxxxxliil** [comment]
> I'm not agree with this PR.
> You can add volumes as you want to get the log files.

However, the access_log for nginx and php-fpm cannot be viewed with existing configuration files
That's why I created this PR

**Comment by Magicalex** [comment]
You can create all the volumes you want. I don't understand.

**Comment by xxxxxliil** [comment]
> You can create all the volumes you want. I don't understand.

At the moment, if I don't go into the container and change the nginx file, I can only see the nginx error_log, but most of the time I need to see the nginx access_log

**Comment by xxxxxliil** [comment]
> You can create all the volumes you want. I don't understand.



> You can create all the volumes you want. I don't understand.

Also, /tmp is not an ideal place to store log files

**Comment by xxxxxliil** [comment]
> > You can create all the volumes you want. I don't understand.
> 
> Also, /tmp is not an ideal place to store log files

In a working container, /tmp has a lot of stuff that has nothing to do with logging, and if I try to add volumes to /tmp, I get a lot of unnecessary stuff

```
/flarum/app # ls /tmp
client_body  fastcgi      nginx.pid    proxy        scgi         uwsgi
```

**Pull request closed without merging**